src/poliastro/constants.py

```diff
@@ -337,3 +378,45 @@
     "Howard Curtis",
     system="si",
 )
+
+critical_inclinations = [
+    Constant(
+        "i1",
+        "first critical inclination for earth's frozen orbit",
+        63.4349 * np.pi / 180,
+        "rad",
+        0,
+        "Orlov, A. A (1953)",
+        system="si",
+    ),
+    Constant(
+        "i2",
+        "second critical inclination for earth's frozen orbit",
+        116.5651 * np.pi / 180,
+        "rad",
+        0,
+        "Orlov, A. A (1953)",
+        system="si",
+    ),
+]
+
+critical_argps = [
```

Comment:
These values are "unimportant enough" that we can hardcode them in the `frozen` code, I think.